include/ardpolyfills/arduino.h

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Arduino.h
```

Comment:
This is the correct file name, but the actual name of this file is `arduino.h`, which is wrong